src/components/App/App.js

```diff
@@ -56,6 +57,10 @@ class App extends Component {
         <Route path='/addtree'>
           <Form addTreeToState={this.addTreeToState} />
         </Route>
+        <Route>
+          <Error error={`Page not found. Click 'Get a Tree' to return to the main page.`} />
```

Comment:
Nice work! love the redirect to the main page when people write in garbage url paths